docs/fedora-releases-guide/index.md

```diff
@@ -253,3 +253,28 @@ you can override this behaviour by specifying
 `allowed_builders` in the [job configuration](/docs/configuration/downstream/bodhi_update).
 
 For a configuration example and retriggering, see [dist-git onboarding](./dist-git-onboarding.md#bodhi-update-job).
+
+
+# Permission Handling
+
+The downstream automation is a chain of jobs: propose downstream/pull from upstream -> koji build -> bodhi update.
+
+This chain can break (resulting in no followup job) if the permissions are not configured correctly in the Packit configuration.
+
+### Propose downstream/Pull from upstream -> Koji build
+
+A Koji build can be triggered by a merged pull request, a direct push or a comment.
+
+* If you only merge Packit created pull requests, you don't need to specify other settings.
+* If you create your own pull request and you want Packit to automatically trigger a Koji build when you merge it you need to be in the [`allowed_pr_authors`](https://packit.dev/docs/configuration/downstream/koji_build#optional-parameters) list, even when you are reusing Packit commits.
+On the other side, if you want to break the chain because, as an example, you want to build the package by yourself in side-tags then you can create a new pull request, re-using the Packit commits, and you have to be sure you are not in the [`allowed_pr_authors`](https://packit.dev/docs/configuration/downstream/koji_build#optional-parameters) list.
```

Comment:
```suggestion
On the other hand, if you want to break the chain because, for example, you want to build the package by yourself in a side-tag then you can create a new pull request containing Packit commits and you have to make sure you are not in the [`allowed_pr_authors`](https://packit.dev/docs/configuration/downstream/koji_build#optional-parameters) list.
```